src/instruments/src/EFB/Components/Form/SimpleInput/SimpleInput.tsx

```diff
@@ -105,7 +114,7 @@ const SimpleInput: FC<SimpleInputProps> = (props) => {
                         <input
                             className={`px-5 py-1.5 text-lg text-gray-300 rounded-lg bg-navy-light border-2 border-navy-light focus-within:outline-none
                             focus-within:border-teal-light-contrast ${props.className}`}
-                            value={displayValue}
+                            value={props.value ?? displayValue}
```

Comment:
Can you explain the reason for this one? If `props.value` changes the useEffect above would update `displayValue`, right? I'm probably missing something.

Reply:
Sorry about this one, I think I was testing something and accidentally forgot to remove it before committing.

Reply:
I do that all the time. 😁